rules/autoconsent/ciaopeople-it.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ciaopeople.it",
+  "runContext": {
+    "urlPattern": "^https://(www\\.)?(ohga|lexplain|wamily|fanpage)\\.it/"
+  },
+  "prehideSelectors": ["#cp-gdpr-choices"],
+  "detectCmp": [
+    { "waitForVisible": "#cp-gdpr-choices" },
+    { "exists": "#cp-gdpr-choices" }
+  ],
+  "detectPopup": [
+    { "waitForVisible": "#cp-gdpr-choices" },
+    { "exists": "#cp-gdpr-choices" }
```

Comment:
Right now `detectCmp` and `detectPopup` sections are identical. Usually this does not make sense. If the visibility of `#cp-gdpr-choices` is enough to start the opt-out, how about: we keep the "exists" rule in detectCmp, and use `visible` rule in `detectPopup`.
Note that there's already a retry logic built in, so explicit wait rule is usually not necessary.